models/zeroex/ethereum/zeroex_ethereum_api_fills.sql

```diff
@@ -455,11 +455,11 @@ SELECT
         END AS taker, -- fix the user masked by ProxyContract issue
         taker_token,
         ts.symbol AS taker_symbol,
-        maker_token,
+        maker_token, 
         ms.symbol AS maker_symbol,
         CASE WHEN lower(ts.symbol) > lower(ms.symbol) THEN concat(ms.symbol, '-', ts.symbol) ELSE concat(ts.symbol, '-', ms.symbol) END AS token_pair,
         taker_token_amount_raw / pow(10, tp.decimals) AS taker_token_amount,
-        taker_token_amount_raw,
+        cast(taker_token_amount_raw as varchar(42)) as taker_token_amount_raw,
```

Comment:
any particular reason this column is explicitly casted, but `maker` one isn't? also, other blockchains aren't. the column is already string in the db

Reply:
ran intto datatype error earlier, issues seems to be resolved now